### ISSUE TITLE
Avalanche activity and snowpit pages - small changes

### ISSUE THREAD

**Issue by amarienthal** [jimurl/gnfac-d6-d8#331]
1. On the accordion 'snapshot' of each event on the Avalanche Activity and Incident pages, change link to more details to "More Avalanche Details". "Incident details" is too specific.

2. Accidents on main menu should be labelled "Accident Reports".

3. On Incidents and Activity pages, change the icon for images to more generic, not instagram logo. Examples: https://www.google.com/search?q=image+icon&rlz=1C1CHBF_enUS795US795&tbm=isch&tbo=u&source=univ&sa=X&ved=2ahUKEwim1rH3mMXeAhXFSt8KHc8zBiYQ7Al6BAgGEB8&biw=1280&bih=833

4. On main navigation menu, when I mouse over "More" A info bubble pops up ("All about the GNFAC and Friends of GNFAC..."), and it blocks the first or two menu items. Maybe get rid of that.

5. Links on snowpit page (and a few other places) are broken:
Link to SWAG at AAA is now: https://www.americanavalancheassociation.org/swag/
Link to NAC is now: https://avalanche.org//national-avalanche-center/

Automatic hyperlinks to glossary terms like 'stability' go to: https://staging.avalanche.org/avalanche-education/stability, which is locked.
The NAC encyclopedia is now located here: https://avalanche.org/avalanche-encylopedia/#stability
Maybe we also link to that from our "Glossary" instead of our page.?

**Comment by amarienthal** [comment]
We decided yes to make the link from "Glossary terms" on the main menu to go to the avalanche.org/avalanche-encyclopedia/

**Comment by jimurl** [comment]
OK, I took care of number 1-5. And updated the links I could find to SWAG on AAA and NAC.

And, glossary terms : I fixed the link to their website ( it has much much better layout, esp. for mobile, than before!) But that final thing about linking to their glossary, they don't have a universal glossary page that includes a list of everything; it defaults to the first term in the list, 'anchor-trees or bushes sticking up... ' 
I couldn't find a comprehensive list of all glossary terms on the NAC site, but I suggest we just improve our own glossary listing. Right now, it is just the brief popup messages that appear in the text, on hover. 
We could fill it in with all the text from the NAC encyclopedia site, auto. We could display it like the avalanche activity pages, with an accordion.
